examples/vae/README.md

```diff
@@ -0,0 +1,9 @@
+# Basic VAE Example
```

Comment:
Do you have some example images or a test set ELBO that this model achieves?

Reply:
I added a generated image and ELBO of a test set (mean and std for 5 trials)